### PR TITLE
Add: Card Component

### PR DESCRIPTION
**Pull request by Tarandeep-s1ngh**
1. Add: Card Component
2. Add: Documentation for Card Component
3. Variations:
- Card with Badge
- Card with Text Overlay
- Horizontal Card
- Text only Card
- Card with Dismiss
- Card with Shadow

Note: not fully responsive yet.

image
image
image
image
image

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]

**Comment by rutvikpumak** [comment]
Please increase the font size of the actual price.

File: Components/card/card.html [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Card | Blaze UI</title>
+
+    <link href="/css/shared.css" rel="stylesheet">
+    <link href="/css/documentation.css" rel="stylesheet">
```
Comment (rutvikpumak) [inline Components/card/card.html:11]: Can u please merge this all css file into one and then add only single link into HTML.

File: Components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    border-radius: 10px;
+}
+
+
+/*=============================================
+=            Card with Dismiss            =
+=============================================*/
+
+.card-dismiss {
+    background-color: white;
```
Comment (rutvikpumak) [inline Components/card/card.css:184]: use variable here like u use for black as well on 193

File: Components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    display: flex;
+    flex-flow: column wrap;
+    text-align: left;
+}
+
+.card-header-txt {
+    padding: 0 1rem;
+}
+
+.card-header-txt h3{
```
Comment (ScoobyDubeyDo) [inline Components/card/card.css]: Give a class to this element

File: Components/card/card.css [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+    background: white;
+    width: 100%;
+    max-width: 31.25rem;
+    margin: 1rem 0;
+    border-radius: 5px;
+    align-self: flex-start;
+    cursor: pointer;
+}
+
+.card-horizontal .img-in-horizontal img{
```
Comment (ScoobyDubeyDo) [inline Components/card/card.css]: Use universal selector here in place of `img` tag.

**Review by ScoobyDubeyDo — APPROVED** [review]

File: Components/card/card.css (hunk 3 above)
Comment (Tarandeep-s1ngh) [reply Components/card/card.css]: Made the required changes! ✨

File: Components/card/card.css (hunk 4 above)
Comment (Tarandeep-s1ngh) [reply Components/card/card.css]: Made the required changes! ✨

**Pull request merged**